webpack/components/Content/ContentPage.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Col, Row, Form, FormGroup } from 'react-bootstrap';
+import Search from '../../components/Search/index';
+import ContentTable from './ContentTable';
+
+const ContentPage = ({
+  content, tableSchema, onSearch,
+  getAutoCompleteParams, updateSearchQuery,
+  initialInputValue, onPaginationChange,
+}) => (
+  <Grid bsClass="container-fluid">
+    <Row>
+      <Col sm={12}>
+        <h1>{__('Ansible Collections')}</h1>
+      </Col>
+    </Row>
+    <Row>
+      <Col sm={6}>
+        <Form className="toolbar-pf-actions">
+          <FormGroup className="toolbar-pf toolbar-pf-filter">
+            <Search
+              onSearch={onSearch}
+              getAutoCompleteParams={getAutoCompleteParams}
+              updateSearchQuery={updateSearchQuery}
+              initialInputValue={initialInputValue}
+            />
+          </FormGroup>
+        </Form>
+      </Col>
+    </Row>
+    <Row>
+      <Col sm={12}>
+        <ContentTable
+          content={content}
+          tableSchema={tableSchema}
+          onPaginationChange={onPaginationChange}
+        />
+      </Col>
+    </Row>
+  </Grid>
+);
+
+ContentPage.propTypes = {
+  content: PropTypes.shape({}).isRequired,
+  tableSchema: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
+  onSearch: PropTypes.func.isRequired,
+  getAutoCompleteParams: PropTypes.func.isRequired,
+  updateSearchQuery: PropTypes.func.isRequired,
+  initialInputValue: PropTypes.string.isRequired,
+  onPaginationChange: PropTypes.func.isRequired,
+};
+
+export default ContentPage;
```

Comment:
@johnpmitsch : Does this look ok to you? We have a component ContentPage corresponding to the main list page for the content..I will work on the details page if this looks like a good place to start..

Reply:
At a glance, that looks fine to me!